internal/builder/builder_test.go

```diff
@@ -318,11 +318,7 @@ func testBuilder(t *testing.T, when spec.G, it spec.S) {
 				defer os.RemoveAll(tmpDir)
 
 				layerFile := filepath.Join(tmpDir, "order.tar")
-				f, err := os.Create(layerFile)
-				h.AssertNil(t, err)
-				defer f.Close()
-
-				err = archive.CreateSingleFileTar(f.Name(), "/cnb/order.toml", "some content")
+				err = archive.CreateSingleFileTar(layerFile, "/cnb/order.toml", "some content")
```

Comment:
I had thought of that too. I chose to leave it using the `archive` variant to keep the test doing exactly what it did before, until a need arose to use the image-os-aware variant in `layers`. I think there's definitely an argument for doing that eventually, but currently nothing in the tests are requiring it (since we're currently using fake images that don't actually have to write Windows-specific layers).